src/js/node/net.js

```diff
@@ -706,6 +706,7 @@ class Server extends EventEmitter {
   [bunSocketServerConnections] = 0;
   [bunSocketServerOptions];
   maxConnections = 0;
+  listening = false;
```

Comment:
on nodejs this is a getter, so you can do something like:
```suggestion
  get listening() {
      return !!this.#server;
  }
```